todo_app/app.py

```diff
@@ -1,40 +1,35 @@
 from flask import Flask, redirect, render_template, request
-from todo_app.data.session_items import add_item, delete_item, get_item, get_items, save_item
+from todo_app.data.trello_items import create_card, delete_card, get_all_cards, increment_complete_card
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
 @app.route('/')
 def index():
-    itemsToDisplay = sorted(get_items(), key=lambda item: item['status'], reverse=True)
-
+    itemsToDisplay = get_all_cards()
+    
     return render_template('index.html', items=itemsToDisplay)
 
 @app.route('/add', methods=['POST'])
 def addNewItem():
     newItemTitle = request.form['addItemFormTitle']
 
-    if newItemTitle == '':
+    if newItemTitle != '':
+        create_card(newItemTitle)
         return redirect('/')
-    
-    add_item(newItemTitle)
-
-    return redirect('/')
+        
+    return
```

Comment:
This now just errors because it doesn't return anything - is that what you want? Returning an explicit error, even if it was just the below, might be nicer:
```suggestion
    return "You didn't specify a title!", 400
```